apps/server/src/modules/files-storage/api/controller/api-test/files-storage-preview.api.spec.ts

```diff
@@ -195,7 +194,7 @@ describe('File Controller (API) - preview', () => {
 						.query(defaultQueryParameters);
 					const result = response.body as ApiValidationError;
 
-					expect(result.message).toEqual('The requested FileRecord: [object Object] has not been found.');
+					expect(result.message).toEqual('The requested FileRecordEntity: [object Object] has not been found.');
```

Comment:
[object Object] sieht danach aus als ob dort das object nicht in JSON umgewandelt wurde.